### PR TITLE
Integrate order statistics API at tickets route

### LINKED ISSUE #671: Integrate order statistics API on tickets route
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

https://user-images.githubusercontent.com/13910561/29230756-f5dabe20-7f01-11e7-8c5f-58ee69c38e65.png

Will work on this.

### PR DESCRIPTION
**Pull request by harshitagupta30**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
The PR integrated the order statistics API at events/view/tickets route.

Fixes #671 
@fossasia/open-event-frontend Please review!

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-765-open-event-frontend-fossasia.surge.sh

**Comment by sumedh123** [comment]
@harshitagupta30 @CosmicCoder96 I am getting 500 at tickets route.

**Comment by niranjan94** [comment]
Deployment link: https://pr-765-open-event-frontend-fossasia.surge.sh

**Comment by CosmicCoder96** [comment]
@sumedh123 It's working fine for me

**Comment by harshitagupta30** [comment]
@niranjan94 @geekyd @sumedh123 please review.

File: app/templates/events/view/tickets/index.hbs [hunk 1]
```diff
@@ -95,11 +95,11 @@
       <tr>
         {{#link-to 'events.view.tickets.orders.index' tagName='td'
         class='ui green inverted segment link'}}
-          <span>{{t 'Completed'}}</span>
```
Comment (geekyd) [inline app/templates/events/view/tickets/index.hbs]: @harshitagupta30 Can you confirm this from the server guys? Will there be stats for `completed` or not ?

Comment (harshitagupta30) [inline app/templates/events/view/tickets/index.hbs]: @geekyd right now the API doesn't support completed in the response. So I guess we can't have it here. If needed we can, later on, add it. I guess it good to merge now. 
@niranjan94 @CosmicCoder96 @sumedh123 Please review.

Comment (sumedh123) [inline app/templates/events/view/tickets/index.hbs]: @geekyd @harshitagupta30 But that's what the main part is. @geekyd What do you think? Rest all looks good to me.

Comment (niranjan94) [inline app/templates/events/view/tickets/index.hbs]: @harshitagupta30 completed is necessary. Please open a server issue.

Comment (harshitagupta30) [inline app/templates/events/view/tickets/index.hbs]: @niranjan94 Opened a issue in the server https://github.com/fossasia/open-event-orga-server/issues/4410

**Comment by niranjan94** [comment]
Deployment link: https://pr-765-open-event-frontend-fossasia.surge.sh

**Comment by harshitagupta30** [comment]
@niranjan94 I have added completed here. I was not sure if `draft` attribute has to be placed here or not and if yes then in which color? Please let me know.

**Comment by harshitagupta30** [comment]
@niranjan94 Can you please guide? Thanks.

**Comment by harshitagupta30** [comment]
@niranjan94 Can you please guide and let me know if `drafts` is required or not? Thanks.

**Comment by niranjan94** [comment]
@harshitagupta30 logically ... How could there be a `draft` state for a ticket/order ?

**Comment by harshitagupta30** [comment]
@niranjan94 okay! I have added made the other changes and added `completed`. Please review.

**Comment by niranjan94** [comment]
Deployment link: https://pr-765-open-event-frontend-fossasia.surge.sh

**Pull request merged**